Download from HuggingFace with private token (#390)
metatensor/metatrain: docs/src/getting-started/checkpoints.rst

    mtt export experimental.soap_bpnn https://my.url.com/model.ckpt --output model.pt

> (Downloading private models is also supported, only through HuggingFace and using the
> ``--huggingface_api_token`` flag.)

Reviewer: ```suggestion
Downloading private HuggingFace models is also supported, by specifying the corresponding API token with the ``--huggingface_api_token`` flag.
```
